spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/metrics/web/reactive/server/MetricsWebFilter.java

```diff
@@ -109,7 +109,8 @@ private void record(ServerWebExchange exchange, Throwable cause, long start) {
 			Iterable<Tag> tags = this.tagsProvider.httpRequestTags(exchange, cause);
 			long duration = System.nanoTime() - start;
 			AutoTimer.apply(this.autoTimer, this.metricName, annotations,
-					(builder) -> builder.tags(tags).register(this.registry).record(duration, TimeUnit.NANOSECONDS));
+					(builder) -> builder.description("Duration of requests made to HTTP Server").tags(tags)
```

Comment:
I wonder if something like "Duration of HTTP server request handling" would be better here? To me, it makes it clearer that the metric is for a server handling a request from a client rather than a client making a request to a server.

Reply:
Yeah, definitely I agree.